### ISSUE TITLE
Polymode and Rmd: many ess functions do not work

### ISSUE THREAD

**Issue by dakni** [vspinu/polymode#157]
I am running GNU Emacs 26.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30) of 2018-07-05 on Manjaro and use polymode-20170307.1122. My ESS version is ess-20180726.2049

After upgrading to Emacs 26 many of the other nice features of ESS do not work anymore in an Rmd document:
- tab completion of R objects 
- auto-complete
- ess-insert-assign
- help on functions in minibuffer after typing some function [it is only shown when the pointer is directly on the function but not after the opening `(`]

There are no messages or anything so what might be the reason for this, rather unfortunate, behavior?

`C-Enter` still works to send commands

Could anyone guide me how I could debug this?

**Comment by dakni** [comment]
on Arch Linux with the same setup I receive constantly these errors (but perhaps these are not related to this issue?!)

```lisp
error in process sentinel: ess-r--flymake-check-errors: Invalid function: flymake-log
error in process sentinel: Invalid function: flymake-log
```

**Comment by vspinu** [comment]
Yes. Things broke badly on emacs26. Time permitting, I will fix this and other issues in the following days.
